spec/models/project_log_spec.rb

```diff
@@ -1,9 +1,12 @@
 require 'rails_helper'
 
 describe ProjectLog, :type => :model do
-  before { pending 're-implement in new project' }
-  it_should_behave_like 'translatable', ['Explanation', 'DetailedExplanation']
-  it_should_behave_like 'mediable'
```

Comment:
When you update this to take care of the merge conflicts, etc, you should be able to put this line back, but with `media_attachable` since that exists and is merged in now.